src/cli.js

```diff
@@ -17,13 +17,15 @@ prog
     .option('--dryrun', 'Simulates changes')
     .option('--forceWait', 'Forces waiting for migrations that do not strictly manage async calls')
     .option('--quiet', 'disables console debug logging within fireway\'s migrate')
+    .option('--stats', 'Displays migration stats')
```

Comment:
Why is it here called `--stats` but below `--showstats`? (I like `--stats`)

Reply:
it was stats before but there is already a variable with stats in the code. However, I will just rename stats to showstats in that function. That should work